css-pseudo-4/Overview.bs

```diff
@@ -936,6 +936,49 @@ Cascading and Per-Element Highlight Styles</h3>
   css/css-pseudo/cascade-highlight-002.html
   </wpt>
 
+<h4 id="highlight-custom-properties">
+Custom Properties in Highlight Styles</h3>
+
+  A [=custom property=] used in highlight styles uses the
+  first custom property value found in the highlight cascade.
+  However, if a custom property value cannot be resolved
+  through the highlight cascade, the property value is taken from
+  the root element.
```

Comment:
Instead of this, I would take this sentence:

>  Additionally, for highlight pseudo-elements originating from the root element, the inherited value of color is currentColor, not the initial value.

Split it out into a list item like this:

> Additionally, for [=highlight pseudo-elements=] originating from the [=root element=]:
> * the inherited value of 'color' is ''currentColor'', not the [=initial value=].
> * the value of [=custom properties=] inherit from the [=root element=].